client/app/review/create/(components)/(Form)/Form.tsx

```diff
@@ -0,0 +1,85 @@
+'use client';
+
+import { useState } from 'react';
+
+import { NewReview } from 'types/new_review';
+import TextArea from '@components/UI/TextArea';
+
+import styles from './Form.module.scss';
+import Dropdown from '@components/UI/Dropdown';
+import { Option } from 'react-dropdown';
+
+async function sendFormData(newReview: NewReview) {
+    await fetch('http://localhost:8000/api/review/create', {
```

Comment:
Don't hardcode the localhost section, either store this in a global variable or the .env file.

```ts
// src/config.ts

interface Config {
    QutReviewApiUrl: string
}

const dev = {
    // just have these as empty strings as we don't have a staging or prod api just yet
    QutReviewApiUrl: "https://staging.api"
}

const prod = {
    // just have these as empty strings as we don't have a staging or prod api just yet
    QutReviewApiUrl: "https://prod.api"
}

const config = process.env.REACT_APP_STAGE === "prod" ? prod : process.env.REACT_APP_STAGE === "dev" ? dev : {}

export default {
    QutReviewApiUrl: "http://localhost:8000",
    ...config
} as Config;
```

```ts
// package.json

"scripts": {
    "start": "cross-env REACT_APP_STAGE=local && react-scripts start",
    "build": "cross-env REACT_APP_STAGE=local && react-scripts build",
    "build:dev": "cross-env REACT_APP_STAGE=dev && react-scripts start",
    "build:prod": "cross-env REACT_APP_STAGE=prod && react-scripts build",
    ...
}
```

Reply:
Great suggestion